src/ApiManagement/ApiManagement.ServiceManagement/Models/PsApiManagementRepresentation.cs

```diff
@@ -20,6 +20,8 @@ public class PsApiManagementRepresentation
     {
         public string ContentType { get; set; }
 
+        [CmdletParameterBreakingChange("Sample", ReplaceMentCmdletParameterName = "Example")]
+        [Parameter(Mandatory = false)]
```

Comment:
breaking change warning should be added to cmdlets (not model), you might want to find cmdlets which reference PsApiManagementRepresentation

Reply:
updated, please take a look again